coops-ui/tests/unit/App.spec.ts

```diff
@@ -188,22 +189,44 @@ describe('App', () => {
       .returns(new Promise((resolve) => resolve({
         data:
           {
-            'mailingAddress': {
-              'streetAddress': '1012 Douglas St',
-              'addressCity': 'Victoria',
-              'addressRegion': 'BC',
-              'addressType': 'mailing',
-              'postalCode': 'V8W 2C3',
-              'addressCountry': 'CA'
-            },
-            'deliveryAddress': {
-              'streetAddress': '220 Buchanan St',
-              'addressCity': 'Glasgow',
-              'addressRegion': 'Scotland',
-              'addressType': 'delivery',
-              'postalCode': 'G1 2FFF',
-              'addressCountry': 'UK'
-            }
+            'registeredOffice':
+              {
+                'mailingAddress': {
+                  'streetAddress': '1012 Douglas St',
+                  'addressCity': 'Victoria',
+                  'addressRegion': 'BC',
+                  'addressType': 'mailing',
+                  'postalCode': 'V8W 2C3',
+                  'addressCountry': 'CA'
+                },
+                'deliveryAddress': {
+                  'streetAddress': '220 Buchanan St',
+                  'addressCity': 'Glasgow',
+                  'addressRegion': 'Scotland',
+                  'addressType': 'delivery',
+                  'postalCode': 'G1 2FFF',
+                  'addressCountry': 'UK'
+                }
+              },
+            'recordsOffice':
```

Comment:
This isn't correct for the subject business (CP0001867). I suggest you create a new test/section for BCorp and then test both address records.

Reply:
Sounds good, will do!

Reply:
Done, thanks.